pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -388,7 +388,7 @@ def conv_sequence(
         modules.append(("conv_seq", conv_seq))
         return SequentialWithOutChannels(OrderedDict(modules))
 
-    use_noise = not impl_params
+    use_noise = not impl_params or not instance_norm
```

Comment:
1. Could you add links here?
2. Does that mean, that in the paper introducing `instance_norm` the additional noise is not used / mentioned at all?

Reply:
In the master branch the input is only filled with noise in `mode='texture'` otherwise it is ignored.

Reply:
[L16-L18](https://github.com/pmeier/texture_nets/blob/aad2cc6f8a998fedc77b64bdcfe1e2884aa0fb3e/models/pyramid.lua#L16-L18)

Reply:
For clarity I would prefer:

```suggestion
    use_noise = impl_params and not instance_norm
```